include/config.h

```diff
@@ -59,6 +59,8 @@
 #define WIFI_SSID_MAX_LEN 64
 #define WIFI_KEY_MAX_LEN 64
 
+bool atob(const char *a);
```

Comment:
This was intended to be a private helper for atob used in Config.cpp, but making it visible here so it can be used in console.cpp is a decent idea for now. Originally it was from console.cpp, so I was confused. No changes, just a comment.